includes/Form.class.php

```diff
@@ -121,6 +121,9 @@ function embed($params) {
 			},
 			success: function(data) {
 				\$j('#form_result_message').html(data.message);
+				var result_class = (data.success) ? 'form_result_success' : 'form_result_error';
+				\$j('#form_result_message').removeClass();

```

Comment:
we sure we want to remove _all classes_?

instead we can do $J(selector).removeClass("form_result_success form_result_error").addClass(result_class); // does the same, but doesn't remove any extra classes